QuantConnect.BybitBrokerage/Models/BybitTrade.cs

```diff
@@ -50,18 +50,6 @@ public class BybitTrade
     /// </summary>
     public decimal? OrderPrice { get; set; }
 
-    /// <summary>
-    /// Order quantity
-    /// </summary>
-    [JsonProperty("orderQty")]
-    public decimal? OrderQuantity { get; set; }
-
-    /// <summary>
-    /// Remaining order quantity
-    /// </summary>
-    [JsonProperty("leavesQty")]
-    public decimal? QuantityRemaining { get; set; }
```

Comment:
Hey @arodus!
Removed `leavesQty/orderQty` because they are not supported for spot trading, so replaced them by keeping track of the accumulative filled quantity, what do you think? Left a todo, we need to review whether fees should be taken into account when checking the expected total fill quantity in all cases
<img width="629" alt="image" src="https://github.com/QuantConnect/Lean.Brokerages.ByBit/assets/18473240/e0006ce1-b31d-4c5b-a25e-19ec403a7969">

Was a bit confused by the objective of `HandleOrderUpdate`. I saw market orders getting execution reports with partially filled events and finally getting a cancel status back from the brokerage which would be consumed by `HandleOrderUpdate` with status `partiallyfilledcanceled` 🤔 

Also pushed a few tweaks here and there after some testing, appreciate your input👍

Reply:
I was, or actually still am quite confused by this too. For orders with partial fills, it often happens (at least on testnet) that the order is not fully filled as bybit is trying to protect the buyer when the spread is getting to big. But I'm not sure if this is the only cause, I've also seen it a few times when the remaining quantity was very small. 
I haven't seen it on the live net so far but I also wasn't testing with huge position sizes there.

Reply:
> For Market Order: If the est. filled price deviates by more than a certain percentage from the last traded price, the market buy order will be partially executed. Any remainder of the order exceeding the limit will be canceled. A market order will only be filled within 800 transactions.

ref https://www.bybit.com/en-US/help-center/s/article/Bybit-Spot-Trading-Rules

Right seems to be the case, current implementation seems okay though, emits partial fills until fully filled or cancelled by bybit 👍